### ISSUE TITLE
npm install canvas failed!

### ISSUE THREAD

**Issue by ojlovecd** [Automattic/node-canvas#572]
Hi, all! I am going to install the `canvas` module for node.js on my OS X. I followed [this post](https://github.com/Automattic/node-canvas/wiki/Installation---OSX)  . Every things were fine until the `Installing node-canvas` step. I got errors below:

```
> canvas@1.2.3 install /Users/Jo/node_modules/canvas
> node-gyp rebuild

  SOLINK_MODULE(target) Release/canvas-postbuild.node
  SOLINK_MODULE(target) Release/canvas-postbuild.node: Finished
  CXX(target) Release/obj.target/canvas/src/Canvas.o
In file included from ../src/Canvas.cc:7:
../src/Canvas.h:19:10: fatal error: 'cairo/cairo.h' file not found
#include <cairo/cairo.h>
         ^
1 error generated.
make: *** [Release/obj.target/canvas/src/Canvas.o] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:269:23)
gyp ERR! stack     at ChildProcess.emit (events.js:110:17)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:1074:12)
gyp ERR! System Darwin 14.3.0
gyp ERR! command "node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/Jo/node_modules/canvas
gyp ERR! node -v v0.12.4
gyp ERR! node-gyp -v v1.0.3
gyp ERR! not ok 
npm ERR! Darwin 14.3.0
npm ERR! argv "node" "/usr/local/bin/npm" "install" "canvas"
npm ERR! node v0.12.4
npm ERR! npm  v2.10.1
npm ERR! code ELIFECYCLE

npm ERR! canvas@1.2.3 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the canvas@1.2.3 install script 'node-gyp rebuild'.
npm ERR! This is most likely a problem with the canvas package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls canvas
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/Jo/npm-debug.log
```

I know someone have already encountered similar problems, and I have tried every solutions I could find(including [255](https://github.com/Automattic/node-canvas/issues/225) and [391](https://github.com/Automattic/node-canvas/issues/391)), but still no luck. I was stuck for two days. I hope someone can help me out! Thanks a lot!

**Comment by LinusU** [comment]
@ojlovecd Could you test #574 and see if it solves this for you?

**Comment by LinusU** [comment]
This should be fixed in version `1.2.5`, please reopen if problem persists.
